### PR TITLE
Increase readability of item information against bright backgrounds

### PR DESCRIPTION
**Pull request by Asaayu**
This pull request edits the CSS formatting of the inventory to add a semi-transparent black background to the item information container, allowing effortless reading of the text when placed against bright backgrounds.
Before and after images have been provided to show the changes in the presentation to the end-user.

**Before and After Images**
**[Before #1](https://i.imgur.com/TBCYU1G.png) -- [After #1](https://i.imgur.com/hS0Wg4j.png)**
**[Before #2](https://i.imgur.com/Tjoj0Rx.png) -- [After #2](https://i.imgur.com/pgPW6wj.png)**

### CODE REVIEW

**Review by TonybynMp4 — APPROVED** [review]

**Review by ShomyTKD — APPROVED** [review]
Works perfectly on the newest version of qb-inventory! Content is wrapped around in a box with a transparent background that greatly increases readability.

**Review by dexter9006 — APPROVED** [review]

**Review by woerki — APPROVED** [review]
LGTM

**Review by woerki — APPROVED** [review]

File: html/css/main.css [hunk 1]
```diff
@@ -397,6 +397,16 @@ body {
 	height: 23vh;
 }
 
+.iteminfo-content {
+    width: fit-content;
+    float: right;
```
Comment (Z3rio) [inline html/css/main.css:402]:
```suggestion
    float: right;
```
Why is float used? Considering it hasnt needed this previously?

File: html/css/main.css [hunk 2]
```diff
@@ -397,6 +397,16 @@ body {
...
+    background-color: rgba(0, 0, 0, 0.3);
+    border: 1px solid rgb(68 68 68 / 40%);
```
Comment (Z3rio) [inline html/css/main.css:404]:
```suggestion
    border: 1px solid rgba(68, 68, 68, 0.4);
```
Using rgba instead of rgb

**Review by Z3rio — CHANGES_REQUESTED** [review]

**Pull request closed without merging**